### PR TITLE
Problem Solved of Rat in a maze

### PR DESCRIPTION
**Pull request by Rajat-Jain29**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(Write your answer here.)

### CODE REVIEW

File: C-Plus-Plus/cp/Rat in a maze.cpp [hunk 1]
```diff
@@ -0,0 +1,118 @@
+/* C++ program to solve Rat in
```
Comment (HarshCasper) [inline C-Plus-Plus/cp/Rat in a maze.cpp:1]: Describe the Problem in detail. If possible add a Problem Link on some Online Coding Portal.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Code looks great. Maybe @iamrajiv and @atarax665 would like to take a look. 

Make apt README changes as well. And change the file name from `Rat in a maze.cpp` to `rat_in_maze.cpp`

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Rajat-Jain29 Add sample input-output, space and time complexity. Also, update the README file and rename the program to `rat_in_a_maze.cpp `.

**Comment by atarax665** [comment]
@Rajat-Jain29 Are you still working on this PR? Do the suggested changes ASAP.

**Comment by HarshCasper** [comment]
Closing the PR because of no updates.

**Pull request closed without merging**